saged/models.py

```diff
@@ -348,6 +379,65 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
         return x
 
 
+class FiveLayerImputation(nn.Module):
+    """An imputation model based off the FiveLayerImputation model"""
```

Comment:
```suggestion
    """An imputation model based off the DeepClassifier (five-layer) model"""
```
I think this is what you meant, since this class is `FiveLayerImputation`

Reply:
Yep, good catch!